tools/table_annovar/table_annovar.xml

```diff
@@ -42,7 +42,7 @@
         #set operation = ','.join( $operation )
 
         ## Annotate variants.
-        table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
+        \${SCRIPT_PATH}/table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
```

Comment:
> Also, I'm not able to permanently edit the PATH variable, since this is managed from above to protect the other users from themselves.

You only need to append to the PATH in one of the env.sh files, or you append to the PATH when you start galaxy's handlers, this will not affect anything except for the current galaxy process.